File: c++/1046.cpp
#include <iostream>

using namespace std;

int main() {
    int hi, hf, horas;

    cin >> hi >> hf;
    if (hi == hf) {
        horas = 24;
    } else if (hi < hf) {
        horas = hf - hi;
    } else if (hi > hf) {
        horas = 24 - (hi - hf);
    }

    cout << "O JOGO DUROU " << horas << " HORA(S)\n";
    return 0;
}

File: python/1046.py
# -*- coding: utf-8 -*-

n1, n2 = map(int, input().split())

if n1 < n2:
    h = n2 - n1
else:
    h = (24 - n1) + n2

print('O JOGO DUROU {} HORA(S)'.format(h))
